corehq/form_processor/migrations/0095_remove_xforminstance_deleted_state.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.20 on 2023-09-13 19:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('form_processor', '0094_add_partial_index_xforms'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='xforminstance',
+            name='state',
+            field=models.PositiveSmallIntegerField(choices=[(1, 'normal'), (2, 'archived'), (4, 'deprecated'), (8, 'duplicate'), (16, 'error'), (32, 'submission_error')], default=1),
+        ),
```

Comment:
Just wondering about this now - since there are records in the DB which will have a different state not represented here, will Django error when loading them?

Reply:
What do you mean by loading them? The state is just an integer value so I think the worst that would happen is that it just won't recognize it as whatever valid state it's checking for and the check will fail (i.e. if form.state == XFormInstance.ARCHIVED). I did another look around that's seems to be the only usage of the state value. Though this does remind me that all soft_deleted forms will now be in the ARCHIVED state until it's permanently deleted, which shouldn't be an issue but I'm wondering if there's some other aspect to that I haven't thought about.